projects/ng-habanero/src/lib/components/Divider/index.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'hb-divider',
```

Comment:
is the selector how it will be referenced in template code? `<hb-divider>` for example

Reply:
Yep, the angular cli wants a prefix